Maya/cicd/batch/buildMaya2Babylon.bat

```diff
@@ -0,0 +1,54 @@
+@echo off
```

Comment:
Hey, I'm not having much luck with this build script, currently the solution fails to build on your branch using this script, but builds in Visual Studio without issue.

Reply:
Hey, and I have added command line arguments to the batch scripts which will allow different versions of Maya. I do not, however, have any version other than 2021 to test with, and am unsure if versions previous of python will support pytest (etc).
Please try and run the command as follows, and let me know if you continue to have problems:
(using an administrator cmd shell and within the Exporters\Maya directory)
cicd\batch\buildMaya2Babylon.bat MAYAVERSION 
cicd\batch\testMaya2Babylon.bat MAYAVERSION

Reply:
Make sure you have added the msbuild.exe directory to system %PATH% as well. It is located in the visual studio install directory and is required to build solution files from a cmd shell.

Reply:
Thanks, I must have had my path setup to use the wrong MSBuild, I also noticed I needed to install `pip` for my mayapy distribution in order to run the test pass.